CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 **Please Note**: This change log only covers v1.3 forwards - apologies for anything missing prior to that.
 
+## Version 5.0.0
```

Comment:
We're not going to just call it version 2? :-P

Reply:
why would we do such a thing???